src/pages/saksbehandling/steg/formue/utils.ts

```diff
@@ -0,0 +1,84 @@
+import { Nullable } from '~lib/types';
+import { Behandlingsinformasjon, FormueStatus, Verdier } from '~types/Behandlingsinformasjon';
+import { SøknadInnhold } from '~types/Søknad';
+
+export function kalkulerFormue(verdier: Nullable<Verdier>) {
+    if (!verdier) {
+        return 0;
+    }
+
+    const formuer = [
+        verdier.verdiIkkePrimærbolig,
+        verdier.verdiKjøretøy,
+        verdier.innskudd,
+        verdier.verdipapir,
+        verdier.pengerSkyldt,
+        verdier.kontanter,
+    ].filter(Boolean) as number[];
+
+    return formuer.reduce((acc, formue) => acc + formue, 0) - (verdier.depositumskonto ?? 0);
+}
+
+export function totalVerdiKjøretøy(
+    kjøretøyArray: Nullable<Array<{ verdiPåKjøretøy: number; kjøretøyDeEier: string }>>
+) {
+    if (kjøretøyArray === null) {
+        return 0;
+    }
+
+    return kjøretøyArray.reduce((acc, kjøretøy) => acc + kjøretøy.verdiPåKjøretøy, 0);
+}
+
+export function kalkulerFormueFraSøknad(f: SøknadInnhold['formue']) {
+    return (
+        [
+            f.verdiPåBolig ?? 0,
+            f.verdiPåEiendom ?? 0,
+            totalVerdiKjøretøy(f.kjøretøy),
+            f.innskuddsBeløp ?? 0,
+            f.verdipapirBeløp ?? 0,
+            f.skylderNoenMegPengerBeløp ?? 0,
+            f.kontanterBeløp ?? 0,
+        ].reduce((acc, formue) => acc + formue, 0) - (f.depositumsBeløp ?? 0)
+    );
+}
+
+export function setInitialValues(behandlingsInfo: Behandlingsinformasjon, søknadsInnhold: SøknadInnhold) {
```

Comment:
Jeg syns navngivingen her var tricky. For det første er det forvirrende at vi har `setInitialValues` og `setInitialVerdier`, og for det andre er det rart at de heter noe med `set`; såvidt jeg kan se ligner de mer på `get`-operasjoner.

Reply:
Det är egentligen en konvertering från backend-strukturen till frontend-strukturen, så både get och set är lite rart. Skjer egentligen på lite andra städer också, så vi borde nog ha en struktur för det.

Reply:
Tänker att både set/get kanske är lite rart. Vad med initializeXXX eller noe?`

Reply:
nvm hehehe